cfme/tests/candu/test_vm_graph.py

```diff
@@ -43,18 +54,29 @@ def test_vm_most_recent_hour_graph_screen(graph_type, provider, enable_candu):
         graph = view.vm_cpu_state
     elif graph_type == 'vm_memory':
         graph = view.vm_memory
+    elif graph_type == 'vm_disk':
+        graph = view.vm_disk
     else:
         graph = view.vm_network
 
     assert graph.is_displayed
 
+    def refresh():
+        provider.browser.refresh()
+        view = navigate_to(vm, 'candu')
+        view.options.interval.fill('Most Recent Hour')
+
+    # wait, some time graph took time to load
+    wait_for(lambda: len(graph.all_legends) > 0,
+             delay=5, timeout=600, fail_func=refresh)
+
+    graph_data = 0
     for leg in graph.all_legends:
         graph.hide_legends(leg)
         assert not graph.legend_is_displayed(leg)
         graph.display_legends(leg)
         assert graph.legend_is_displayed(leg)
 
-        leg_data = 0
         for data in graph.data_for_legends(leg).values():
```

Comment:
**OPTIONAL:** Could you add a comment explaining what you are trying to achieve with this part. Also, perhaps explain why the resulting sum should be greater than zero.